chess/backend/src/move/move.py

```diff
@@ -38,14 +41,16 @@ def set_initial(self, game: Games):
         self.post_move_fen = INITIAL_FEN
         self.move_algebraic = ""
         self.game_id = game
-        self.turn = WHITE
+        # We set the initial turn to black, so that the server will know that the
```

Comment:
When is this called? What if the player is black and the AI is white?

Reply:
In the `set_state_from_prev_move` function, it take a `Moves` database object (that represents the previous move) and sets the state for the current, new move. It says the new move color is the opposite of the previous moves color. 

Now, white always moves first in chess. So whoever is white, either computer or AI, will load the initial move from the moves database, see that their initial move was black, and then set the first real move of the game to the opposite of black, which is white.

Reply:
Nice